fixtures/socialaccount_socialapp.json

```diff
@@ -26,33 +26,5 @@
     "key": "", 
     "provider": "google"
   }
-},
-{
-  "pk": 3, 
-  "model": "socialaccount.socialapp", 
```

Comment:
@viliambalaz Este pozor na to, ze fixtures sa loaduju iba pri vytvarani DB. Aby sa zmazane `socialapp` odstranili aj na produkcii, treba spravit aj datovu migraciu, ktora ich z DB zmaze. Napriklad teraz, ked spustis `setup.py`, tak stale to pyta client ID a secret aj pre twitter a linkedin, kedze su stale v DB.

Reply:
@martinmacko47 
1. Ked na produkcii existuje len jeden user s twitter uctom, nemoze SocialApp vymazat rucne admin?
2. FK(`null=False`) na `socialapp` maju modely `socialaccount_socialapp_sites` a `socialtoken`. Ale predsa ak vymazem instanciu `socialapp` tak sa implicitne vymazu aj instancie `socialaccount_socialapp_sites` a `socialtoken`. Preco mam potom vymazat aj suvisiace veci?
3. Treba vymazat aj profily vytvorene cez tieto socialne siete?
4. V akom priecinku mam vytvorit tuto data-migraciu? Subor `env/.../allauth/socialaccount/` nemame na git repozitary.

Reply:
5. Treba vytvorit aj "backwards" migraciu, v ktorej sa `socialapp` "Linekdin OAuth", "Twitter OAuth" znova vytvoria?

Reply:
k 1.: Moze, ale je dobrym zvykom, aby strukturalne zmeny v DB robili automaticky migracie. Aby sme mali istotu, ze sa spravia spravne. Admin by mal robit iba ukony, ktore nemenia logiku. Je trochu nestastne na allauth, ze na pridanie/odobranie providera treba upravit aj `INSTALLED_APPS` aj zmenit obsah DB. Takze ani admin nemoze samostatne pridat/odobrat providera, ani kod bez migracie.

k 2.: Nepozeral som, ako su tam tie zavislosti. Ak je FK taky, ze sa kaskadovo zmazu suvisiace veci, tak to nemusis riesit explicitne. Ciel je, aby po zbehnuti migracii nebola v DB zmienka o odstranenych login appkach.

k 3.: Nie, userov nemazeme, iba im mazeme login cez tieto socialne siete. Stale sa mozu prihlasit emailom a heslom.

k 4.: Dobra otazka. Musime ju spravit v niektorej nasej app. Asi najvhodnejsia bude `chcemvediet/apps/accounts`. Vid aj https://stackoverflow.com/questions/29520929/django-datamigration-for-external-app

k 5.: Idealne ano.